training/instructlab/Makefile

```diff
@@ -42,19 +42,19 @@ intel: instructlab
 	"${CONTAINER_TOOL}" build --squash-all -t oci:../build/instructlab-$@  -f instructlab/containers/hpu/Containerfile instructlab
 
 .PHONY: nvidia-quay
-nvidia: instructlab
+nvidia-quay: instructlab
 	rm -rf ../build/instructlab-$@
 	"${CONTAINER_TOOL}" build --squash-all -t quay.io/ai-lab/instructlab-nvidia:latest instructlab/containers/cuda
 	"${CONTAINER_TOOL}" push quay.io/ai-lab/instructlab-nvidia:latest
```

Comment:
Do we want to include the pushes? Not everyone will have permissions to do this, they would need an owner quay account or access to the bot. Plus this means people are going to end up pushing their own builds rather than doing it through CI (despite CI being broken ATM)

Reply:
This was added in a previous commit. I just fixed the target names which were breaking the build (overrides the main build targets unintentionally)

I assume it was added for a Konflux pipeline but I agree the hardcoded destination limits its use. It should probably be changed to a variable image name but then wouldnt be much different than directly invoking the push command.

Reply:
Theres a separate pr for that, see: #584